packages/annotorious-react/package.json

```diff
@@ -8,7 +8,8 @@
   "funding": "https://steadyhq.com/rainer-simon",
   "type": "module",
   "scripts": {
-    "start": "vite",
+    "start-image": "cd test/image && vite --host",
+    "start-openseadragon": "cd test/openseadragon && vite --host",
```

Comment:
Just a single `vite` picks the last `index.html` under the `test` folder hierarchy. Effectively running only the `openseadragon` test